src/main/scala/org/apache/spark/deploy/armada/submit/ArmadaClientApplication.scala

```diff
@@ -1110,17 +1180,7 @@ private[spark] class ArmadaClientApplication extends SparkApplication {
       )
   }
 
-  /** Resolves a value based on precedence: CLI > Template > Default
-    *
-    * @param cliValue
```

Comment:
Removed as it was a comment pointing to something very obvious and self-explanatory.